land_claims.civmap.json

```diff
@@ -68,9 +68,10 @@
 {"name":"Adina\n(Exclave,Caledonia)","shortname":"Adina","color":"#15316a","declutter":3,"x":6900,"z":3700,"id":"bdf09a53-1de3-4be9-a1ff-2a54dedd83c5","polygon":[[[6708,3412],[6706,3372],[6634,3394],[6590,3412],[6528,3454],[6496,3474],[6472,3556],[6466,3612],[6506,3626],[6492,3746],[6516,3784],[6562,3804],[6600,3758],[6624,3714],[6638,3676],[6678,3656],[6718,3678],[6738,3698],[6758,3728],[6742,3758],[6706,3766],[6682,3776],[6706,3812],[6726,3854],[6772,3854],[6796,3880],[6822,3902],[6836,3924],[6882,3910],[6882,3876],[6904,3868],[6932,3882],[6956,3928],[6972,3934],[6988,3939],[7072,3852],[7091,3826],[7093,3795],[7145,3780],[7090,3705],[7144,3658],[7158,3605],[7194,3592],[7275,3564],[7272,3499],[7140,3440],[7073,3425],[7009,3432],[6967,3417],[6940,3439],[6938,3483],[6879,3514],[6848,3480],[6800,3514],[6811,3545],[6780,3570],[6680,3471],[6687,3428],[6708,3412]]]},
+{"name":"Albion Listening Post\n(Antipodean Union)","shortname":"(AU)","color":"#002c80","id":"e8a04487-f6b2-4f88-8543-5a920ea1b416","polygon":[[[-7490,3365],[-7590,3365],[-7590,3400],[-7490,3400]]]},
```

Comment:
I moved this one down on purpose because the last entries render on top of the first entries and this one conflicts 100% with commonwealth, so by putting it below commonwealth it's still clickable whereas here it's not